mobile/src/bundles/common/types/navigation/root-navigation-parameter-list.type.ts

```diff
@@ -1,9 +1,17 @@
-import { type RootScreenName } from '~/bundles/common/enums/enums';
+import {
+    type RootScreenName,
+    type TalentOnboardingScreenName,
+} from '~/bundles/common/enums/enums';
 
 type RootNavigationParameterList = {
     [RootScreenName.AUTH_ROOT_ROUTE]: undefined;
     [RootScreenName.MAIN_ROOT_ROUTE]: undefined;
     [RootScreenName.ONBOARDING_ROOT_ROUTE]: undefined;
+    [TalentOnboardingScreenName.PROFILE]: undefined;
+    [TalentOnboardingScreenName.BSA_BADGES]: undefined;
+    [TalentOnboardingScreenName.SKILLS_AND_PROJECTS]: undefined;
+    [TalentOnboardingScreenName.CV_AND_CONTACTS]: undefined;
+    [TalentOnboardingScreenName.PREVIEW]: undefined;
 };
```

Comment:
We have this already in talent-onboarding-navigation-parameter-list-type.ts and others

Reply:
fix